features/performance/manifests/07-mc-pre-boot-worker-tuning.yaml

```diff
@@ -0,0 +1,42 @@
b29rcy9wcmUtdWRldi8KICAgIG1rZGlyIC1wIC4vZXRjL3N5c3RlbWQvCiAgICBta2RpciAtcCAuL2V0Yy9zeXNjb25maWcvCiAgICB0b3VjaCAuL2V0Yy9zeXN0ZW1kL3N5c3RlbS5jb25mCiAgICB0b3VjaCAuL2V0Yy9zeXNjb25maWcvaXJxYmFsYW5jZQogICAgdG91Y2ggLi91c3IvbGliL2RyYWN1dC9ob29rcy9wcmUtdWRldi8wMC10dW5lZC1wcmUtdWRldi5zaAogICAgY2htb2QgK3ggLi91c3IvbGliL2RyYWN1dC9ob29rcy9wcmUtdWRldi8wMC10dW5lZC1wcmUtdWRldi5zaAoKICAgIGdldF9jcHVfbWFzayAxCiAgICBlY2hvICcjIS9iaW4vc2gKCiAgICB0eXBlIGdldGFyZ3MgPi9kZXYvbnVsbCAyPiYxIHx8IC4gL2xpYi9kcmFjdXQtbGliLnNoCgogICAgI2NwdW1hc2s9IiQoZ2V0YXJncyBub25faXNvX2NwdW1hc2spIgogICAgY3B1bWFzaz0nJG5vbl9pc29fY3B1bWFzaycKCiAgICBsb2coKQogICAgewogICAgZWNobyAidHVuZWQ6ICRAIiA+PiAvZGV2L2ttc2cKICAgIH0KCiAgICBpZiBbIC1uICIkY3B1bWFzayIgXTsgdGhlbgogICAgZm9yIGZpbGUgaW4gL3N5cy9kZXZpY2VzL3ZpcnR1YWwvd29ya3F1ZXVlL2NwdW1hc2sgL3N5cy9idXMvd29ya3F1ZXVlL2RldmljZXMvd3JpdGViYWNrL2NwdW1hc2s7IGRvCiAgICAgICAgbG9nICJzZXR0aW5nICRmaWxlIENQVSBtYXNrIHRvICRjcHVtYXNrIgogICAgICAgIGlmICEgZWNobyAkY3B1bWFzayA+ICRmaWxlIDI+L2Rldi9udWxsOyB0aGVuCiAgICAgICAgbG9nICJFUlJPUjogY291bGQgbm90IHdyaXRlIENQVSBtYXNrIGZvciAkZmlsZSIKICAgICAgICBmaQogICAgZG9uZQogICAgZmknID4gLi91c3IvbGliL2RyYWN1dC9ob29rcy9wcmUtdWRldi8wMC10dW5lZC1wcmUtdWRldi5zaAoKICAgICMgU2V0IENQVSBhZmZpbml0eSBhY2NvcmRpbmcgdG8gcmVzZXJ2ZWRfY3B1cwogICAgZ2V0X2NwdV9hZmZpbml0eQogICAgZWNobyAiW01hbmFnZXJdIiA+PiAuL2V0Yy9zeXN0ZW1kL3N5c3RlbS5jb25mCiAgICBlY2hvICJDUFVBZmZpbml0eT0kY3B1X2FmZmluaXR5IiA+PiAuL2V0Yy9zeXN0ZW1kL3N5c3RlbS5jb25mCgogICAgIyBTZXQgSVJRIGJhbm5lZCBjcHUgYWNjb3JkaW5nIHRvIHJlc2VydmVkX2NwdXMKICAgIGdldF9jcHVfbWFzayAwCiAgICBlY2hvICJJUlFCQUxBTkNFX0JBTk5FRF9DUFVTPSRub25faXNvX2NwdW1hc2siID4+IC4vZXRjL3N5c2NvbmZpZy9pcnFiYWxhbmNlCiAgICAKICAgIGZpbmQgLiB8IGNwaW8gLWNvID4uLi9pc29faW5pdHJkLmltZwogICAgY2QgLi4KICAgICMgVE9ETyAtIGZpbmQgYSBtb3JlIHJvYnVzdCBhcHByb2FjaCB0aGFuIGtlZXBpbmcgdGhlIGxhc3QgdGltZXN0YW1wCiAgICBSSENPU19PU1RSRUVfUEFUSD0kKGxzIC10ZCAvYm9vdC9vc3RyZWUvKi8gfCBoZWFkIC0xKQogICAgY3AgaXNvX2luaXRyZC5pbWcgJFJIQ09TX09TVFJFRV9QQVRICiAgICBSSENPU19PU1RSRUVfUEFUSD0ke1JIQ09TX09TVFJFRV9QQVRIIyIvYm9vdCJ9CgogICAgIyBHZXQgY3VycmVudCBvc3RyZWUgY29uZmlnIGZpbGUgYWNjb3JkaW5nIHRvIHRoZSBsYXRlc3QgdmVyc2lvbgogICAgY3VycmVudF92ZXI9MQogICAgZW50cnlfZmlsZT0kKGxzIC10ZCAvYm9vdC9sb2FkZXIvZW50cmllcy8qIHwgaGVhZCAtMSkKICAgIHdoaWxlIHJlYWQgLXIgbGluZSA7IGRvCiAgICAgICAgdmVyPWBhd2sgJy92ZXJzaW9uLyB7cHJpbnQgJDJ9JyAkbGluZWAKICAgICAgICBpZiBbICIkdmVyIiAtZ3QgIiRjdXJyZW50X3ZlciIgXTsgdGhlbgogICAgICAgICAgIGN1cnJlbnRfdmVyPSR2ZXIKICAgICAgICAgICBlbnRyeV9maWxlPSRsaW5lCiAgICAgICAgZmkKICAgIGRvbmUgPDw8JChlZ3JlcCAkKHVuYW1lIC1yKSAtbHIgL2Jvb3QvbG9hZGVyL2VudHJpZXMvKQoKICAgIHNlZCAtaSAic15pbml0cmQgLipcJF4mICR7UkhDT1NfT1NUUkVFX1BBVEh9aXNvX2luaXRyZC5pbWdeIiAkZW50cnlfZmlsZQoKICAgICNUT0RPIC0gb25jZSBSSENPUyBpbWFnZSBjb250YWlucyB0aGUgaW5pdHJkIGNvbnRlbnQgd2UgY2FuIHNldCBwYXJhbWV0ZXJzIHdpdGggcnBtLW9zdHJlZToKICAgICNycG0tb3N0cmVlIGluaXRyYW1mcyAtLWVuYWJsZSAtLWFyZz0tSSAtLWFyZz0vZXRjL3N5c3RlbWQvc3lzdGVtLmNvbmYKICAgICNycG0tb3N0cmVlIGluaXRyYW1mcyAtLWVuYWJsZSAtLWFyZz0tSSAtLWFyZz0vZXRjL3N5c2NvbmZpZy9pcnFiYWxhbmNlCiAgICAKICAgIHN5c3RlbWN0bCByZWJvb3QKZmkK
```

Comment:
@yanirq @MarSik how do we plan to do patch reviews for these blobs?

Reply:
@simon3z  this is a generated manifest,  maybe it should'nt be commit.
The generation should align with @slintes work on https://github.com/openshift-kni/baremetal-deploy/pull/43

Reply:
We can keep it here as a reference / example as in: https://github.com/openshift-kni/baremetal-deploy/blob/22c85c79b537743fbbaa143425244e117749da09/features/rt/manifests/06-mc-rtkernel-worker-rt.yaml#L23